.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: c
+dist: trusty   # Travis build machine is Ubuntu Trusty 14.04 x64
+sudo: required # A sudo enabled, full VM is used for building instead of a docker container
+
+git:
+  submodules: false
+
+before_install:
+  - sudo apt-get update -qq
+  # Pre-reqs for building omsagent
+  - sudo apt-get install -qq git pkg-config make g++ rpm librpm-dev libpam0g-dev libssl-dev libkrb5-dev curl libcurl4-gnutls-dev  
+
+# We are in /home/travis/build/Microsoft/Powershell-DSC-for-Linux/
+
+
+install: 
+  # Clone OMI
+  - cd ..
```

Comment:
I'd be more inclined to use the superproject since you have one. That makes this .yml file more robust (you know the superproject will work due to nightly builds, this may not if the project is reorganized).

Also, you'll miss other files (like OMI version file, etc).